Bottom: d028f7e8aed6687e53512dfd8a0acd347e0e8b34
Top:    48e008a226119948df95bda2046812bb8b23d9d5
Author: Alexandre Oliva <aoliva@redhat.com>
Date:   2016-02-26 21:28:43 -0300

Refresh of adjust-copyright-notice.patch

---

diff --git a/libcc1/libcp1plugin.cc b/libcc1/libcp1plugin.cc
index d0cfaf2..32f9702 100644
--- a/libcc1/libcp1plugin.cc
+++ b/libcc1/libcp1plugin.cc
@@ -2,7 +2,7 @@
    Copyright (C) 2014-2016 Free Software Foundation, Inc.
 
    This file is part of GCC.  As it interacts with GDB through libcc1,
-   they all become a single program.
+   they all become a single program as regards the GNU GPL's requirements.
 
    GCC is free software; you can redistribute it and/or modify it under
    the terms of the GNU General Public License as published by the Free
